src/test/java/tr/com/srdc/cda2fhir/IntegrationTest.java

```diff
@@ -0,0 +1,167 @@
+	       
+	        return bundle;
+	 }
+	 
+	
+	 
+	private static void validate(Bundle bundle) {
+		FhirValidator validator = new FhirValidator(ctx);
+    	ValidationResult result = validator.validateWithResult(bundle);
+    	if(!result.isSuccessful()) {
+    		System.out.println(result.toString());
+    	}
+    	Assert.assertTrue(result.isSuccessful());
+	}
+    
+	private Map<String, String> getResourceProfileMap(Bundle bundle) {
+		HashMap<String, String> map = new HashMap<String, String>();
+		
+		for( BundleEntryComponent entry : bundle.getEntry()) {
+			if(entry != null) {
+				map.put(entry.getResource().getResourceType().name() ,"");
+			}
+		}
+		
+		return map;
+	}
+
+	private String rakiaPatientIntegration(Bundle bundle) throws Exception {
+		
+		List<Patient> patients = bundle.getEntry().stream()
+				.map(r -> r.getResource())
+				.filter(r -> (r instanceof Patient))
+				.map(r -> (Patient) r)
+				.collect(Collectors.toList());
+		
+    	Patient patient = patients.get(0);
+    	
+    	MethodOutcome outcome = client.create()
+ 			   .resource(patient)
+ 			   .prettyPrint()
+ 			   .encodedJson()
+ 			   .execute();
+    	
+    	Assert.assertTrue(outcome.getCreated());
+    	return outcome.getResource().getIdElement().getIdPart();
+	}
+	
+	
+    @Test
+	public void rakiaIntegration() throws Exception {
+    	String sourceName = "Cerner/Person-RAKIA_TEST_DOC00001 (1).XML";
+    	// generate bundle with patient
+		Bundle ccdaBundle  = generateCcdaBundle(sourceName);
+		
+		// currently doesn't valiate. Potentially due to incorrect implementation
+		// of resourceProfileMap.
+		// TODO: Make valid.
+		// validate(patientBundle);
+		
+		// Generate empty resourceProfileMap to appease transaction generator function
+		// TODO: properly implement getResouceProfileMap
+		Map<String, String> resourceProfileMap = getResourceProfileMap(ccdaBundle);
+
+    	// create transaction bundle from ccda bundle
+    	Bundle transactionBundle = generateTransactionBundle(ccdaBundle, resourceProfileMap); 
+    	// print pre-post bundle
+    	FHIRUtil.printJSON(transactionBundle, "src/test/resources/output/rakia_bundle.json");
+    	
+    	// Send transaction bundle to server.
+    	Bundle resp = client.transaction().withBundle(transactionBundle).execute();
+    	
+    	for(BundleEntryComponent entry : resp.getEntry()) {
+    		BundleEntryResponseComponent entryResp =entry.getResponse();
+    		
+    		Assert.assertEquals("201 Created", entryResp.getStatus());
+    	}
+    	
+    	// TODO: Test each entry exists in server via search
+//    	for( BundleEntryComponent entry : transactionBundle.getEntry()) {
+//    		System.out.println(entry.getClass());
+//    		Bundle results = (Bundle) client
+//    	    	      .search()
+//    	    	      .forResource((Class<? extends IBaseResource>) entry.getClass())
+//    	    	      .prettyPrint()
+//    	    	      .execute();
+//    		
+//    		Assert.assertTrue(results.hasEntry());
+//    	}
```

Comment:
Are you going to do some type of check here? I think we need something; not complicated but something.